### PR TITLE
Feat/ux first attempt

### LINKED ISSUE #95: Key issues on GridGenerator
Warning: Each child in a list should have a unique "key" prop.

Check the render method of `GridGenerator`. See https://reactjs.org/link/warning-keys for more information.
    at http://192.168.8.114:4000/static/js/2.chunk.js:14504:23
    at GridGenerator (http://192.168.8.114:4000/static/js/1.chunk.js:30:5)
    at div
    at http://192.168.8.114:4000/static/js/2.chunk.js:19990:27
    at div
    at Transition (http://192.168.8.114:4000/static/js/2.chunk.js:57042:30)
    at http://192.168.8.114:4000/static/js/2.chunk.js:14605:22
    at http://192.168.8.114:4000/static/js/2.chunk.js:13029:23
    at div
    at http://192.168.8.114:4000/static/js/2.chunk.js:13689:23
    at div
    at http://192.168.8.114:4000/static/js/2.chunk.js:12976:96
    at TeamList (http://192.168.8.114:4000/static/js/4.chunk.js:177:5)
    at div
    at http://192.168.8.114:4000/static/js/2.chunk.js:14504:23
    at div
    at http://192.168.8.114:4000/static/js/2.chunk.js:18605:23
    at div
    at http://192.168.8.114:4000/static/js/2.chunk.js:14701:23
    at ParticipateSession (http://192.168.8.114:4000/static/js/4.chunk.js:658:75)
    at Route (http://192.168.8.114:4000/static/js/2.chunk.js:56347:29)
    at AppRoutes (http://192.168.8.114:4000/static/js/main.chunk.js:2299:14)
    at Switch (http://192.168.8.114:4000/static/js/2.chunk.js:56549:29)
    at Suspense
    at Router (http://192.168.8.114:4000/static/js/2.chunk.js:55982:30)
    at BrowserRouter (http://192.168.8.114:4000/static/js/2.chunk.js:55602:35)
    at div
    at http://192.168.8.114:4000/static/js/2.chunk.js:14701:23
    at main
    at ErrorBoundary (http://192.168.8.114:4000/static/js/main.chunk.js:1473:5)
    at AuthProvider (http://192.168.8.114:4000/static/js/main.chunk.js:4210:3)
    at div
    at App (http://192.168.8.114:4000/static/js/main.chunk.js:488:75)
    at I18nextProvider (http://192.168.8.114:4000/static/js/2.chunk.js:51505:19)

### LINKED ISSUE #97: Improve members view
As a member
> I want to be able to see my submitted 
> I want to submit length of time that took me to estimate
> I want to submit my connectivity status (online | offline) if possible 

As an admin 
I want to calculate general max time, max time accurate answer, min time accurate answer, min time general answer, average time for general answer, average time for accurate answer, deviations from values (accurate and general)

### PR DESCRIPTION
**Pull request by carlospatinos**
UX layout in place, first attempt

### CODE REVIEW

**Comment by carlospatinos** [comment]
This solves part of 
https://github.com/carlospatinos/scrum/issues/97

**Comment by carlospatinos** [comment]
Not sure this will be needed, since grid will be removed:
Fix:  https://github.com/carlospatinos/scrum/issues/95

File: packages/scrum-client/src/App.css [hunk 1]
```diff
@@ -1,22 +1,12 @@
 @import url('https://fonts.googleapis.com/css?family=Fira+Sans:400,500,600,700,800');
 
-* {
-  box-sizing: border-box;
-}
-
-h1,h2,h3,h4,h5,h6,label,span {
```
Comment (doncesarts) [inline packages/scrum-client/src/App.css:7]: I think we should create our own bootstrap theme

File: packages/scrum-client/src/App.css [hunk 2]
```diff
@@ -1,22 +1,12 @@
...
-  font-weight: 500;
-  font-family: 'Fira Sans', sans-serif;
-}
-
 .App {
-  /* text-align: center; */
-  margin-top: 15px;
+  max-height: calc(100% - 120px);
+  overflow-y: scroll;
+  padding: 0px 10% !important;
```
Comment (doncesarts) [inline packages/scrum-client/src/App.css]: Why !important;? 
It usually shadows errors or bad levels of specifty on the css selector.

Comment (doncesarts) [reply packages/scrum-client/src/App.css]: is the same margin-top for all devices?

File: packages/scrum-client/src/components/Header.css [hunk 3]
```diff
@@ -0,0 +1,9 @@
+.indigo {
+    background-color: #3f51b5 !important;
```
Comment (doncesarts) [inline packages/scrum-client/src/components/Header.css]: Why important ?

File: packages/scrum-client/src/components/Header.js [hunk 4]
```diff
@@ -13,20 +15,93 @@ export default function Header() {
...
 
   return (
     <header>
       <div>
-        <Navbar bg="light" expand="lg">
-          <Navbar.Brand href={PATHS.HOME}>Scrum</Navbar.Brand>
+        <Navbar bg="dark" variant="dark" className="indigo fixed-top navbar-expand-md" expand="lg">
+          <Navbar.Brand className="" href={PATHS.HOME}>
+            <svg viewBox="0 0 841.9 595.3" style={{ height: '2.5rem', width: '2.5rem' }}>
+              <g fill="#61DAFB">
```
Comment (doncesarts) [inline packages/scrum-client/src/components/Header.js]: this object should be its own asset.  and just import it here 
**svg**

**Comment by doncesarts** [comment]
There a couple of errors on this branch : 
When I load the scrum landing page, i can see scroll bars horizontal and vertical  in my laptop screen. 
When I click on Config a session => Click on Create Session button=> Then it loads scrum landing page instead of the sharing session page.

File: packages/scrum-client/src/App.css (hunk 2 above)
Comment (carlospatinos) [reply packages/scrum-client/src/App.css]: Removed the important part, as i explained, this was more down to the lack of knowledge I had 
On the margins, i dont know, I may need some help on that and also regarding the scrolling. Ill create a issue and assign that to you if you can help me. I dont have much experience on css to the level you may have :D 

Issue #100 (https://github.com/carlospatinos/scrum/issues/100)

File: packages/scrum-client/src/components/Header.css (hunk 3 above)
Comment (carlospatinos) [reply packages/scrum-client/src/components/Header.css]: removed

File: packages/scrum-client/src/App.css (hunk 1 above)
Comment (carlospatinos) [reply packages/scrum-client/src/App.css:7]: Thats a great idea 🥇

**Pull request merged**